src/config.ts

```diff
@@ -150,6 +150,7 @@ export type HlsConfig = {
   enableSoftwareAES: boolean;
   minAutoBitrate: number;
   loader: { new (confg: HlsConfig): Loader<LoaderContext> };
+  fetchSetup?: (context: any, initParams: any) => Request;
```

Comment:
```suggestion
  fetchSetup?: (context: LoaderContext, initParams: any) => Request;
```

matches the types that are used where this is actually called